eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/PropertiesUtils.java

```diff
@@ -61,4 +62,14 @@ public static Properties getPropertiesByPrefix(final Properties from, final Prop
         );
         return to;
     }
+
+    public static Properties readPropertiesFile(String fileName) {
+        try (final InputStream inputStream = PropertiesUtils.class.getClassLoader().getResourceAsStream(fileName)) {
+            Properties properties = new Properties();
+            properties.load(inputStream);
+            return properties;
+        } catch (Exception e) {
+            throw new IllegalArgumentException(String.format("File: %s is not exist", fileName));
```

Comment:
Is it inappropriate to throw an IllegalArgumentException here?

Reply:
done